apis/logging/v1/output_types.go

```diff
@@ -225,3 +228,24 @@ type Loki struct {
 	// +optional
 	LabelKeys []string `json:"labelKeys,omitempty"`
 }
+
+// Splunk Deliver log data to Splunk’s HTTP Event Collector
+// Provides optional extra properties for `type: splunk_hec` ('splunk_hec_logs' after Vector 0.23
+type Splunk struct {
+	// The name of the index where to send the events to. If not specified, the default index is used.
+	// Note: This parameter supports Vector's template syntax, which enables you to use dynamic per-event values.
+	//
+	// +optional
+	Index string `json:"index,omitempty"`
```

Comment:
@alanconway do we wish to expose this to admins?  When we say "default" in our notes is this whatever their default is or do we intend to use dynamic expansion to set the value to something like what we do for elasticsearch?  (e.g. app/infra/audit)

Reply:
fixed